diff -Naur Makefile ./Makefile
--- Makefile	2005-05-24 19:46:16.000000000 -0700
+++ Makefile	2012-09-07 21:53:41.349756454 -0700
@@ -1,24 +1,21 @@
 SUBDIR=lib bin etc script include
+
 PREFIX= /usr/local/v6eval
-DOCFILES=00README INSTALL CHANGELOG \
-	 00README.v6eval INSTALL.v6eval CHANGELOG.v6eval \
-	 COPYRIGHT
+FILES=	00README INSTALL CHANGELOG \
+	00README.v6eval INSTALL.v6eval CHANGELOG.v6eval \
+	COPYRIGHT
 
-DIRS=   $(PREFIX) $(PREFIX)/bin $(PREFIX)/man $(PREFIX)/doc
-#SECTIONS=       1 2 3 4 5 6 7 8 9 n
-SECTIONS=       1
+DIRS=   $(PREFIX) $(PREFIX)/bin $(PREFIX)/man/man1 ${FILESDIR}
 .MAIN: depend all
-beforeinstall:
-	for i in $(DIRS); do \
-		[ -d $$i ] || mkdir $$i; \
-	done
-#	for i in $(SECTIONS); do \
-#		[ -d $(PREFIX)/man/man$$i ] || mkdir $(PREFIX)/man/man$$i; \
-#		[ -d $(PREFIX)/man/cat$$i ] || mkdir $(PREFIX)/man/cat$$i; \
-#	done
-	for i in $(SECTIONS); do \
-		[ -d $(PREFIX)/man/man$$i ] || mkdir $(PREFIX)/man/man$$i; \
-	done
-	install -c -o bin -g bin -m 444 $(DOCFILES) $(PREFIX)/doc
+
+.for d in ${DIRS}
+${d}:
+	${MKDIR} ${d}
+beforeinstall: ${d}
+.endfor
+
+afterinstall:
+	cd ${.CURDIR}; \
+	    ${INSTALL} -o bin -g bin -m 444 ${FILES} ${DESTDIR}${PREFIX}/doc
 
 .include <bsd.subdir.mk>
diff -Naur Makefile.inc ./Makefile.inc
--- Makefile.inc	1969-12-31 16:00:00.000000000 -0800
+++ Makefile.inc	2012-09-07 21:23:58.968754290 -0700
@@ -0,0 +1,5 @@
+LIBDIR?=	/usr/local/lib
+
+LIBCM?=	${DESTDIR}${LIBDIR}/libCm.a
+LIBPZ?=	${DESTDIR}${LIBDIR}/libPz.a
+LIBPKT?=${DESTDIR}${LIBDIR}/libpkt.a
diff -Naur bin/Makefile.inc ./bin/Makefile.inc
--- bin/Makefile.inc	2002-12-02 04:31:27.000000000 -0800
+++ bin/Makefile.inc	2012-09-07 21:11:53.985755045 -0700
@@ -1,14 +1,16 @@
 BINDIR?=	/usr/local/v6eval/bin
-MANDIR=	/usr/local/v6eval/man/man
-l=../../lib
-Cm=$l/Cm
-Pz=$l/Pz
-pkt=$l/pkt
+MANDIR=		/usr/local/v6eval/man/man
+l=${.CURDIR}/../../lib
+o=${.OBJDIR}/../../lib
+.for i in Cm Pz pkt
+${i}=$l/${i}
+${i}.OBJ=$o/${i}
+.endfor
 DEBUG_FLAGS=-g -Wall -Woverloaded-virtual
 .if exists(/usr/local/v6/lib/libinet6.a)
 LDFLAGS	+= -L/usr/local/v6/lib
 LDADD		+= -linet6
 .endif
 
-.MAIN: depend all
-beforedepend: cleandepend
+#.MAIN: depend all
+#beforedepend: cleandepend
diff -Naur bin/checker/Makefile ./bin/checker/Makefile
--- bin/checker/Makefile	2006-08-30 18:57:53.000000000 -0700
+++ bin/checker/Makefile	2012-09-07 21:21:30.528754550 -0700
@@ -1,14 +1,8 @@
-CXXFLAGS+=	-I${Cm} -I${Pz} -I${pkt} -I/usr/local/include
-LDFLAGS+=	-L${Cm} -L${Pz} -L${pkt} -L/usr/local/lib
-LDADD+=	-lPkt -lPz -lCm -lcrypto
-SRCS=	checker.cc 
-PROG_CXX=	$(SRCS:.cc=)
-PROG=	${PROG_CXX}
-NOMAN=
+CXXFLAGS+=	-I${Cm.OBJ} -I${Pz.OBJ} -I${pkt.OBJ} -I${Cm} -I${Pz} -I${pkt} -I${INCLUDEDIR}
+LDFLAGS +=	-L${Cm.OBJ} -L${Pz.OBJ} -L${pkt.OBJ}
+LDADD+=		-lPkt -lPz -lCm -lcrypto
+DPADD+=		${LIBCM} ${LIBPZ} ${LIBPKT} ${LIBCRYPTO}
+PROG_CXX=	checker
 NO_MAN=
-CLEANFILES+=	.depend
 
-.include "../Makefile.inc"
-.include "bsd.prog.mk"
-
-$(PROG): ${Cm}/libCm.a ${Pz}/libPz.a ${pkt}/libPkt.a
+.include <bsd.prog.mk>
diff -Naur bin/dhcomp/Makefile ./bin/dhcomp/Makefile
--- bin/dhcomp/Makefile	2006-08-30 18:57:53.000000000 -0700
+++ bin/dhcomp/Makefile	2012-09-07 21:23:38.123754661 -0700
@@ -44,16 +44,13 @@
 #
 ######################################################################
 
-CXXFLAGS+=	-I/usr/local/include
+CXXFLAGS+=	-I${INCLUDEDIR}
 CXXFLAGS+=	-DTAHI
-LDFLAGS +=	-L/usr/local/lib
+LDFLAGS +=	-L${LIBDIR}
 LDADD+=	-lcrypto
-SRCS=	dhcomp.cc 
-PROG_CXX=	$(SRCS:.cc=)
-PROG=	${PROG_CXX}
+DPADD+=	${LIBCRYPTO}
+PROG_CXX=	dhcomp
 NOMAN=
 NO_MAN=
-CLEANFILES+=	.depend
 
-.include "../Makefile.inc"
-.include "bsd.prog.mk"
+.include <bsd.prog.mk>
diff -Naur bin/dhgen/Makefile ./bin/dhgen/Makefile
--- bin/dhgen/Makefile	2006-08-30 18:57:53.000000000 -0700
+++ bin/dhgen/Makefile	2012-09-07 21:23:10.323754873 -0700
@@ -44,16 +44,11 @@
 #
 ######################################################################
 
-CXXFLAGS+=	-I/usr/local/include
+CXXFLAGS+=	-I${INCLUDEDIR}
 CXXFLAGS+=	-DTAHI
-LDFLAGS+=	-L/usr/local/lib
+LDFLAGS+=	-L${LIBDIR}
 LDADD+=	-lcrypto
-SRCS=	dhgen.cc 
-PROG_CXX=	$(SRCS:.cc=)
-PROG=	${PROG_CXX}
-NOMAN=
+PROG_CXX=	dhgen
 NO_MAN=
-CLEANFILES+=	.depend
 
-.include "../Makefile.inc"
-.include "bsd.prog.mk"
+.include <bsd.prog.mk>
diff -Naur bin/file2hex/Makefile ./bin/file2hex/Makefile
--- bin/file2hex/Makefile	2006-08-30 18:57:54.000000000 -0700
+++ bin/file2hex/Makefile	2012-09-07 21:20:06.339754313 -0700
@@ -45,11 +45,8 @@
 ########################################################################
 
 PROG=	file2hex
-SRCS=	file2hex.c
 CFLAGS+=	-Wall -g
 NOMAN=
 NO_MAN=
-CLEANFILES+=	.depend
 
-.include "../Makefile.inc"
-.include "bsd.prog.mk"
+.include <bsd.prog.mk>
diff -Naur bin/ivcomp/Makefile ./bin/ivcomp/Makefile
--- bin/ivcomp/Makefile	2006-08-30 18:57:54.000000000 -0700
+++ bin/ivcomp/Makefile	2012-09-07 21:19:52.293754826 -0700
@@ -45,12 +45,9 @@
 ########################################################################
 
 PROG=	ivcomp
-SRCS=	ivcomp.c
 CFLAGS+=	-Wall -g
 LDADD+=	-lcrypto
-NOMAN=
+DPADD+=	${LIBCRYPTO}
 NO_MAN=
-CLEANFILES+=	.depend
 
-.include "../Makefile.inc"
-.include "bsd.prog.mk"
+.include <bsd.prog.mk>
diff -Naur bin/pktbuf/Makefile ./bin/pktbuf/Makefile
--- bin/pktbuf/Makefile	2006-08-30 18:57:54.000000000 -0700
+++ bin/pktbuf/Makefile	2012-09-07 21:22:57.499754402 -0700
@@ -1,14 +1,8 @@
-CXXFLAGS+=	-I${Cm} -I${Pz} -I${pkt} -I/usr/local/include
-LDFLAGS +=	-L${Cm} -L${Pz} -L${pkt} -L/usr/local/lib
-LDADD+=	-lPkt -lPz -lCm -lcrypto
-SRCS=	pktbuf.cc 
+CXXFLAGS+=	-I${Cm.OBJ} -I${Pz.OBJ} -I${pkt.OBJ} -I${Cm} -I${Pz} -I${pkt} -I/usr/local/include
+LDFLAGS +=	-L${Cm.OBJ} -L${Pz.OBJ} -L${pkt.OBJ}
+LDADD+=		-lPkt -lPz -lCm -lcrypto
+DPADD+=		${LIBCM} ${LIBPZ} ${LIBPKT} ${LIBCRYPTO}
 PROG_CXX=	pktbuf
-PROG=	${PROG_CXX}
-NOMAN=
 NO_MAN=
-CLEANFILES+=	.depend
 
-.include "../Makefile.inc"
-.include "bsd.prog.mk"
-
-$(PROG): ${Cm}/libCm.a ${Pz}/libPz.a ${pkt}/libPkt.a
+.include <bsd.prog.mk>
diff -Naur bin/pktctl/Makefile ./bin/pktctl/Makefile
--- bin/pktctl/Makefile	2006-08-30 18:57:54.000000000 -0700
+++ bin/pktctl/Makefile	2012-09-07 21:22:50.395757018 -0700
@@ -1,12 +1,7 @@
-CXXFLAGS+=	-I${Cm} -I${Pz} -I${pkt} -I/usr/local/include
-LDFLAGS +=	-L${Cm} -L${Pz} -L${pkt} -L/usr/local/lib
-LDADD+=	-lPkt -lPz -lCm -lcrypto
-SRCS=	pktctl.cc 
+CXXFLAGS+=	-I${Cm.OBJ} -I${Pz.OBJ} -I${pkt.OBJ} -I${Cm} -I${Pz} -I${pkt} -I/usr/local/include
+LDFLAGS +=	-L${Cm.OBJ} -L${Pz.OBJ} -L${pkt.OBJ}
+LDADD+=		-lPkt -lPz -lCm -lcrypto
+DPADD+=		${LIBCM} ${LIBPZ} ${LIBPKT} ${LIBCRYPTO}
 PROG_CXX=	pktctl
-PROG=	${PROG_CXX}
-CLEANFILES+=	.depend
 
-.include "../Makefile.inc"
-.include "bsd.prog.mk"
-
-$(PROG): ${Cm}/libCm.a ${Pz}/libPz.a ${pkt}/libPkt.a
+.include <bsd.prog.mk>
diff -Naur bin/pktrecv/Makefile ./bin/pktrecv/Makefile
--- bin/pktrecv/Makefile	2006-08-30 18:57:54.000000000 -0700
+++ bin/pktrecv/Makefile	2012-09-07 21:22:41.437754387 -0700
@@ -1,14 +1,8 @@
-CXXFLAGS+=	-I${Cm} -I${Pz} -I${pkt} -I/usr/local/include
-LDFLAGS +=	-L${Cm} -L${Pz} -L${pkt} -L/usr/local/lib
-LDADD+=	-lPkt -lPz -lCm  -lcrypto
-SRCS=	pktrecv.cc 
+CXXFLAGS+=	-I${Cm.OBJ} -I${Pz.OBJ} -I${pkt.OBJ} -I${Cm} -I${Pz} -I${pkt} -I${INCLUDEDIR}
+LDFLAGS +=	-L${Cm.OBJ} -L${Pz.OBJ} -L${pkt.OBJ}
+LDADD+=		-lPkt -lPz -lCm -lcrypto
+DPADD+=		${LIBCM} ${LIBPZ} ${LIBPKT} ${LIBCRYPTO}
 PROG_CXX=	pktrecv
-PROG=	${PROG_CXX}
-NOMAN=
 NO_MAN=
-CLEANFILES+=	.depend
 
-.include "../Makefile.inc"
-.include "bsd.prog.mk"
-
-$(PROG): ${Cm}/libCm.a ${Pz}/libPz.a ${pkt}/libPkt.a
+.include <bsd.prog.mk>
diff -Naur bin/pktsend/Makefile ./bin/pktsend/Makefile
--- bin/pktsend/Makefile	2006-08-30 18:57:55.000000000 -0700
+++ bin/pktsend/Makefile	2012-09-07 21:22:19.704754725 -0700
@@ -1,12 +1,7 @@
-CXXFLAGS+=	-I${Cm} -I${Pz} -I${pkt} -I/usr/local/include
-LDFLAGS +=	-L${Cm} -L${Pz} -L${pkt} -L/usr/local/lib
-LDADD+=	-lPkt -lPz -lCm  -lcrypto
-SRCS=	pktsend.cc 
+CXXFLAGS+=	-I${Cm.OBJ} -I${Pz.OBJ} -I${pkt.OBJ} -I${Cm} -I${Pz} -I${pkt} -I${INCLUDEDIR}
+LDFLAGS +=	-L${Cm.OBJ} -L${Pz.OBJ} -L${pkt.OBJ}
+LDADD+=		-lPkt -lPz -lCm -lcrypto
+DPADD+=		${LIBCM} ${LIBPZ} ${LIBPKT} ${LIBCRYPTO}
 PROG_CXX=	pktsend
-PROG=	${PROG_CXX}
-CLEANFILES+=	.depend
 
-.include "../Makefile.inc"
-.include "bsd.prog.mk"
-
-$(PROG): ${Cm}/libCm.a ${Pz}/libPz.a ${pkt}/libPkt.a
+.include <bsd.prog.mk>
diff -Naur bin/prfcomp/Makefile ./bin/prfcomp/Makefile
--- bin/prfcomp/Makefile	2006-08-30 18:57:55.000000000 -0700
+++ bin/prfcomp/Makefile	2012-09-07 21:22:03.823754720 -0700
@@ -45,12 +45,9 @@
 ########################################################################
 
 PROG=	prfcomp
-SRCS=	prfcomp.c
 CFLAGS+=	-Wall -g
 LDADD+=	-lcrypto
-NOMAN=
+DPADD+=		${LIBCRYPTO}
 NO_MAN=
-CLEANFILES+=	.depend
 
-.include "../Makefile.inc"
-.include "bsd.prog.mk"
+.include <bsd.prog.mk>
diff -Naur bin/remotes/Makefile ./bin/remotes/Makefile
--- bin/remotes/Makefile	2005-05-24 19:08:48.000000000 -0700
+++ bin/remotes/Makefile	2012-09-07 21:25:56.853754534 -0700
@@ -46,13 +46,18 @@
 SUBDIR=	unknown	\
 	manual
 
-PREFIX=/usr/local/v6eval
-DOCFILES=00README
+FILES=	00README
+FILESDIR=	/usr/local/v6eval
+FILESGRP=	bin
+FILESMODE=	444
+FILESOWN=	bin
+
+INSTALL+=	-d
 
 depend:
 
-afterinstall:
-	install -d -o bin -g bin -m 755 $(PREFIX)/bin
-	install -c -o bin -g bin -m 444 $(DOCFILES) $(PREFIX)/bin/
+#afterinstall:
+#	install -d -o bin -g bin -m 755 $(PREFIX)/bin
+#	install -c -o bin -g bin -m 444 $(DOCFILES) $(PREFIX)/bin/
 
 .include <bsd.subdir.mk>
diff -Naur bin/remotes/manual/Makefile ./bin/remotes/manual/Makefile
--- bin/remotes/manual/Makefile	2009-11-24 23:50:24.000000000 -0800
+++ bin/remotes/manual/Makefile	2012-09-07 20:40:32.826754088 -0700
@@ -104,6 +104,8 @@
 all:
 depend:
 clean:
+cleandir:
+obj:
 
 install:
 	install -d -o bin -g bin -m 755 $(PREFIX)/bin
diff -Naur bin/remotes/unknown/Makefile ./bin/remotes/unknown/Makefile
--- bin/remotes/unknown/Makefile	2005-11-09 19:54:12.000000000 -0800
+++ bin/remotes/unknown/Makefile	2012-09-07 20:40:19.755781294 -0700
@@ -99,6 +99,8 @@
 all:
 depend:
 clean:
+cleandir:
+obj:
 
 install:
 	install -d -o bin -g bin -m 755 $(PREFIX)/bin
diff -Naur bin/sigcomp/Makefile ./bin/sigcomp/Makefile
--- bin/sigcomp/Makefile	2006-08-30 18:57:55.000000000 -0700
+++ bin/sigcomp/Makefile	2012-09-07 21:21:47.005754227 -0700
@@ -45,16 +45,13 @@
 ########################################################################
 
 PROG=	sigcomp
-SRCS=	sigcomp.c
 CFLAGS+=	-Wall -g
 LDADD+=	-lcrypto
-NOMAN=
+DPADD+=		${LIBCRYPTO}
 NO_MAN=
-CLEANFILES+=	.depend
 
 # CFLAGS+=	-DDEBUG
 # LDFLAGS+=	-L/usr/local/lib
 # LDADD+=	-lefence
 
-.include "../Makefile.inc"
-.include "bsd.prog.mk"
+.include <bsd.prog.mk>
diff -Naur bin/x509dec/Makefile ./bin/x509dec/Makefile
--- bin/x509dec/Makefile	2006-08-30 18:57:55.000000000 -0700
+++ bin/x509dec/Makefile	2012-09-07 21:21:41.885754373 -0700
@@ -45,15 +45,12 @@
 ########################################################################
 
 PROG=	x509dec
-SRCS=	x509dec.c
 CFLAGS+=	-Wall -g
 LDADD+=	-lcrypto
-NOMAN=
+DPADD+=		${LIBCRYPTO}
 NO_MAN=
-CLEANFILES+=	.depend
 
 # LDFLAGS+=	-L/usr/local/lib
 # LDADD+=	-lefence
 
-.include "../Makefile.inc"
-.include "bsd.prog.mk"
+.include <bsd.prog.mk>
diff -Naur etc/Makefile ./etc/Makefile
--- etc/Makefile	1999-08-22 17:56:03.000000000 -0700
+++ etc/Makefile	2012-09-07 20:40:45.732758823 -0700
@@ -57,3 +57,6 @@
 
 clean:
 
+cleandir:
+
+obj:
diff -Naur include/Makefile ./include/Makefile
--- include/Makefile	2001-10-04 23:27:30.000000000 -0700
+++ include/Makefile	2012-09-07 20:42:33.060764550 -0700
@@ -59,5 +59,7 @@
 
 depend:
 clean:
+cleandir:
+obj:
 
 #end
diff -Naur lib/Cm/CmMain.cc ./lib/Cm/CmMain.cc
--- lib/Cm/CmMain.cc	2009-08-26 17:10:04.000000000 -0700
+++ lib/Cm/CmMain.cc	2012-08-30 21:58:51.593768336 -0700
@@ -42,19 +42,20 @@
  *
  * $TAHI: v6eval/lib/Cm/CmMain.cc,v 1.19 2009/08/27 00:10:04 akisada Exp $
  */
+#include <sys/param.h>
+#include <sys/time.h>
+#include <sys/resource.h>
 #include "CmMain.h"
 #include <signal.h>
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
 #include <unistd.h>
+#if 0
 #include <utmp.h>
+#endif
 #include <time.h>
 #include <pwd.h>
-#include <sys/time.h>
-#include <sys/types.h>
-#include <sys/resource.h>
-#include <sys/param.h>
 extern int causeStopSignals[];
 static int stopSingals[]={SIGINT,SIGQUIT,SIGTERM,0};
 extern void applicationMain(CmMain*);
@@ -129,6 +130,7 @@
 
 //----------------------------------------------------------------------
 // 障害解析情報作成
+#if 0
 static struct utmp *myUtmpEnt(FILE *in,struct utmp *u) {
 	int s=ttyslot();
 	if(s<0||fseek(in,sizeof(struct utmp)*s,0)<0||
@@ -144,6 +146,7 @@
 	sprintf(p,catch2,A(u->ut_line),A(u->ut_name),A(u->ut_host));
 #undef A
 	return;}
+#endif
 void CmMain::makeCatchEye(const STR pgmName) {
 static char catch1[]="@(#) %02d/%02d/%02d %02d:%02d:%02d %s STARTED by %s@%s";
 	time_t now=time(0); struct tm *m=localtime(&now);
@@ -151,7 +154,10 @@
 	CSTR p=pw?pw->pw_name:"unknown";
 	sprintf(catchStart_,catch1,m->tm_year,m->tm_mon+1,m->tm_mday,
 		m->tm_hour,m->tm_min,m->tm_sec,pgmName,p,myHostname());
-	makeCatch2Eye(catchStart_+strlen(catchStart_));}
+#if 0
+	makeCatch2Eye(catchStart_+strlen(catchStart_));
+#endif
+	}
 
 //----------------------------------------------------------------------
 // 起動時情報
diff -Naur lib/Cm/Makefile ./lib/Cm/Makefile
--- lib/Cm/Makefile	2006-08-30 18:57:56.000000000 -0700
+++ lib/Cm/Makefile	2012-09-07 20:12:50.042085891 -0700
@@ -9,4 +9,4 @@
 CXXFLAGS+=	-I. 
 CLEANFILES+=	.depend
 
-.include "bsd.lib.mk"
+.include <bsd.lib.mk>
diff -Naur lib/Makefile.inc ./lib/Makefile.inc
--- lib/Makefile.inc	2005-01-11 20:41:52.000000000 -0800
+++ lib/Makefile.inc	2012-09-07 20:21:22.355754615 -0700
@@ -1,10 +1,10 @@
 install:
-Cm=../Cm
-Pz=../Pz
+Cm=${.CURDIR}/../Cm
+Pz=${.CURDIR}/../Pz
 NOPROFILE=true
 NO_PROFILE=true
 DEBUG_FLAGS=-g -DYYDEBUG -Wall -Woverloaded-virtual
-YFLAGS  = -dv
+YFLAGS+= -dv
 .y.cc:
 	${YACC} ${YFLAGS} ${.IMPSRC}
 	mv y.tab.c $@
diff -Naur lib/Pz/Makefile ./lib/Pz/Makefile
--- lib/Pz/Makefile	2009-11-24 23:22:17.000000000 -0800
+++ lib/Pz/Makefile	2012-09-07 20:14:11.343754824 -0700
@@ -12,8 +12,8 @@
 	rijndael-alg-fst.c rijndael-api-fst.c				\
 	camellia.c camellia-api.c
 
-CXXFLAGS+=	-O0 -I$(Cm) -I.
-CFLAGS+=	-I$(Cm) -I.
+CXXFLAGS+=	-O0 -I$(Cm) -I${.CURDIR}
+CFLAGS+=	-I$(Cm) -I${.CURDIR}
 
 #CXXFLAGS=	-pipe -g -Wall -I./ -I$(Cm)
 #CFLAGS=	-pipe -g -Wall -I./ -I$(Cm) -I/usr/local/include
@@ -63,4 +63,4 @@
 # 2.2.1. Extended (64-bit) Sequence Number
 # CXXFLAGS+=	-DESN_DBG
 
-.include "bsd.lib.mk"
+.include <bsd.lib.mk>
diff -Naur lib/pkt/Makefile ./lib/pkt/Makefile
--- lib/pkt/Makefile	2006-08-30 18:57:56.000000000 -0700
+++ lib/pkt/Makefile	2012-09-07 20:31:30.325754123 -0700
@@ -1,14 +1,15 @@
+.PATH: ${.CURDIR}
 LIB=	Pkt
 YSRCS=	PzParse.y
+
 SRCS=	$(YSRCS:.y=.cc) LxToken.cc LxLexer.cc RAFControl.cc \
 	PktAgent.cc PktRecv.cc PktClient.cc PktServer.cc StdAgent.cc \
 	BpfAgent.cc Bpfilter.cc Ringbuf.cc RunEnv.cc PktCtlClient.cc \
 	PktSndClient.cc PktRcvClient.cc PktBuf.cc PktQueue.cc bufStat.cc
 
-CXXFLAGS+=	-I. -I$(Cm) -I$(Pz) -DHAVE_BPF_HDRCMPLT -I/usr/include
+CXXFLAGS+=	-I${.CURDIR} -I. -I$(Cm) -I$(Pz) -DHAVE_BPF_HDRCMPLT
 # CXXFLAGS+=	-DVCLEAR_DBG
 
-CLEANFILES+=	PzParse.h
-CLEANFILES+=	.depend
+#CLEANFILES+=	PzParse.h
 
-.include "bsd.lib.mk"
+.include <bsd.lib.mk>
diff -Naur script/pmod/V6evalCommon/Makefile ./script/pmod/V6evalCommon/Makefile
--- script/pmod/V6evalCommon/Makefile	2010-03-17 01:12:29.000000000 -0700
+++ script/pmod/V6evalCommon/Makefile	2012-09-07 20:41:11.644754290 -0700
@@ -9,6 +9,8 @@
 		-e 's/ doc_(perl|site|\$$\(INSTALLDIRS\))_install$$//' \
 		${Makefile}
 
+cleandir:
+
 clean: ${Makefile}
 	${MAKE} -f ${Makefile} realclean
 
@@ -16,3 +18,5 @@
 	${MAKE} -f ${Makefile} install
 
 depend:
+
+obj:
diff -Naur script/pmod/V6evalRemote/Makefile ./script/pmod/V6evalRemote/Makefile
--- script/pmod/V6evalRemote/Makefile	2006-01-16 23:44:01.000000000 -0800
+++ script/pmod/V6evalRemote/Makefile	2012-09-07 20:41:28.255754397 -0700
@@ -9,6 +9,8 @@
 		-e 's/ doc_(perl|site|\$$\(INSTALLDIRS\))_install$$//' \
 		${Makefile}
 
+cleandir:
+
 clean: ${Makefile}
 	${MAKE} -f ${Makefile} realclean
 
@@ -16,3 +18,5 @@
 	${MAKE} -f ${Makefile} install
 
 depend:
+
+obj:
diff -Naur script/pmod/V6evalTool/Makefile ./script/pmod/V6evalTool/Makefile
--- script/pmod/V6evalTool/Makefile	2006-01-16 23:44:01.000000000 -0800
+++ script/pmod/V6evalTool/Makefile	2012-09-07 20:42:18.602754605 -0700
@@ -9,6 +9,8 @@
 		-e 's/ doc_(perl|site|\$$\(INSTALLDIRS\))_install$$//' \
 		${Makefile}
 
+cleandir:
+
 clean: ${Makefile}
 	${MAKE} -f ${Makefile} realclean
 
@@ -16,3 +18,5 @@
 	${MAKE} -f ${Makefile} install
 
 depend:
+
+obj:
